### PR TITLE
[VRGtR] The Undead Warlock, Subclass level indicators

### PR DESCRIPTION
**Pull request by Zencha70**
Changed subclass text info to include the required level milestones & updated file version

### CODE REVIEW

File: supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.1" encoding="utf-8" ?>
```
Comment (NickVendel) [inline supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml:1]: wrong version

**Comment by NickVendel** [comment]
also just in general, that's not how it was done in the book

File: supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml [hunk 2]
```diff
@@ -34,7 +34,7 @@
 	<element name="Expanded Spell List" type="Archetype Feature" source="Van Richten's Guide To Ravenloft" id="ID_WOTC_VRGTR_ARCHETYPE_FEATURE_WARLOCK_UNDEAD_EXPANDED_SPELL_LIST">
 		<supports>Warlock</supports>
 		<description>
-			<p>The Undead lets you choose from an expanded list of spells when you learn a warlock spell. The following spells are added to the warlock spell list for you.</p>
+			<p>At 1st level, the Undead lets you choose from an expanded list of spells when you learn a warlock spell. The following spells are added to the warlock spell list for you.</p>
```
Comment (NickVendel) [inline supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml:37]:
```suggestion
			<p><em>1st-level Undead feature</em></p>
			<p>The Undead lets you choose from an expanded list of spells when you learn a warlock spell. The following spells are added to the warlock spell list for you.</p>
```

File: supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml [hunk 3]
```diff
@@ -64,7 +64,7 @@
 	
 	<element name="Form of Dread" type="Archetype Feature" source="Van Richten's Guide To Ravenloft" id="ID_WOTC_VRGTR_ARCHETYPE_FEATURE_WARLOCK_UNDEAD_FORM_OF_DREAD">
 		<description>
-			<p>You manifest an aspect of your patron’s dreadful power. As a bonus action, you transform for 1 minute. You gain the following benefits while transformed:</p>
+			<p>At 1st level, you manifest an aspect of your patron’s dreadful power. As a bonus action, you transform for 1 minute. You gain the following benefits while transformed:</p>
```
Comment (NickVendel) [inline supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml:67]:
```suggestion
			<p><em>1st-level Undead feature</em></p>
			<p>You manifest an aspect of your patron’s dreadful power. As a bonus action, you transform for 1 minute. You gain the following benefits while transformed:</p>
```

File: supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml [hunk 4]
```diff
@@ -88,7 +88,7 @@
 
 	<element name="Grave Touched" type="Archetype Feature" source="Van Richten's Guide To Ravenloft" id="ID_WOTC_VRGTR_ARCHETYPE_FEATURE_WARLOCK_UNDEAD_GRAVE_TOUCHED">
 		<description>
-			<p>Your patron’s powers have a profound effect on your body and magic. You don’t need to eat, drink, or breathe.</p>
+			<p>At 6th level, your patron’s powers have a profound effect on your body and magic. You don’t need to eat, drink, or breathe.</p>
```
Comment (NickVendel) [inline supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml:91]:
```suggestion
			<p><em>6th-level Undead feature</em></p>
			<p>Your patron’s powers have a profound effect on your body and magic. You don’t need to eat, drink, or breathe.</p>
```

File: supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml [hunk 5]
```diff
@@ -99,7 +99,7 @@
 
 	<element name="Necrotic Husk" type="Archetype Feature" source="Van Richten's Guide To Ravenloft" id="ID_WOTC_VRGTR_ARCHETYPE_FEATURE_WARLOCK_UNDEAD_NECROTIC_HUSK">
 		<description>
-			<p>Your connection to undeath and necrotic energy now saturates your body. You have resistance to necrotic damage. If you are transformed using your Form of Dread, you instead become immune to necrotic damage.</p>
+			<p>At 10th level, your connection to undeath and necrotic energy now saturates your body. You have resistance to necrotic damage. If you are transformed using your Form of Dread, you instead become immune to necrotic damage.</p>
```
Comment (NickVendel) [inline supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml:102]:
```suggestion
			<p><em>10th-level Undead feature</em></p>
			<p>Your connection to undeath and necrotic energy now saturates your body. You have resistance to necrotic damage. If you are transformed using your Form of Dread, you instead become immune to necrotic damage.</p>
```

File: supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml [hunk 6]
```diff
@@ -112,7 +112,7 @@
 
 	<element name="Spirit Projection" type="Archetype Feature" source="Van Richten's Guide To Ravenloft" id="ID_WOTC_VRGTR_ARCHETYPE_FEATURE_WARLOCK_UNDEAD_SPIRIT_PROJECTION">
 		<description>
-			<p>Your spirit can become untethered from your physical form. As an action, you can project your spirit from your body. The body you leave behind is unconscious and in a state of suspended animation.</p>
+			<p>At 14th level, your spirit can become untethered from your physical form. As an action, you can project your spirit from your body. The body you leave behind is unconscious and in a state of suspended animation.</p>
```
Comment (NickVendel) [inline supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml:115]:
```suggestion
			<p><em>14th-level Undead feature</em></p>
			<p>Your spirit can become untethered from your physical form. As an action, you can project your spirit from your body. The body you leave behind is unconscious and in a state of suspended animation.</p>
```

**Review by NickVendel — CHANGES_REQUESTED** [review]

**Review by Zencha70 — COMMENTED** [review]
Looks better and more complete than how I did it, and still quite new at this, so got nothing to add ^^

**Review by NickVendel — APPROVED** [review]

**Review by Azzzzzy — APPROVED** [review]

**Pull request merged**